src/main/resources/application-openshift-metering-worker.yaml

```diff
@@ -0,0 +1,15 @@
+rhsm-subscriptions:
+  # Use Spring Resource notation for this (e.g. "classpath:" or "file:")
+  account-list-resource-location: ${ACCOUNT_LIST_RESOURCE_LOCATION:}
+  account-batch-size: ${ACCOUNT_BATCH_SIZE:1}
+
+  metering:
+    prometheus:
+      metric:
+        openshift:
+          metricPromQL: >-
+            group(subscription_labels{ebs_account='%s', support =~ 'Premium|Standard|Self-Support|None'})by(_id,ebs_account,support)*on(_id)
```

Comment:
@mstead Not sure if this will affect us, but when I use this query against time at which cluster's subscription settings was updated, then I see error as below

`Error executing query: found duplicate series for the match group {_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296"} on the left hand-side of the operation: [{_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296", ebs_account="6661312", support="Standard"}, {_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296", ebs_account="6661312", support="Premium"}];many-to-many matching not allowed: matching labels must be unique on one side`

![Screenshot from 2021-02-08 10-19-20](https://user-images.githubusercontent.com/16951913/107177330-986c3900-69f7-11eb-88e9-2b8764a73440.png)

Reply:
> @mstead Not sure if this will affect us, but when I use this query against time at which cluster's subscription settings was updated, then I see error as below
> 
> `Error executing query: found duplicate series for the match group {_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296"} on the left hand-side of the operation: [{_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296", ebs_account="6661312", support="Standard"}, {_id="6d7a300f-0e78-4f14-90ba-bd2ccfab6296", ebs_account="6661312", support="Premium"}];many-to-many matching not allowed: matching labels must be unique on one side`
> 
> ![Screenshot from 2021-02-08 10-19-20](https://user-images.githubusercontent.com/16951913/107177330-986c3900-69f7-11eb-88e9-2b8764a73440.png)

@ntkathole I'd love for you to show me how to reproduce this. Can you help?

Reply:
Nevermind -- I have a reproducer. :)